### PR TITLE
feat: add pictograms

### LINKED ISSUE #388: Add pictograms library page
A library page should be created for the `@carbon/pictograms` work. It should align with the work done on the icon library sister page.

### PR DESCRIPTION
**Pull request by vpicone**
Closes #388 

- Added Pictograms Library, Usage and Contribution
- Renamed Iconography to Icons to match Pictograms and IDL
- Moved the Library page to be the first tab when the sidebar item is clicked

### CODE REVIEW

**Comment by connor-leech** [comment]
The regex is causing each title to be sentence case even though a fair number of them contain acronyms or capital letters in the second word.

Examples: 
* "IBM z"
* Api

**Comment by vpicone** [comment]
@connor-leech the page uses the `friendly_name` from the `metadata.yml` property in the pictograms [manifest on the carbon repo](https://github.com/carbon-design-system/carbon/blob/81049808150288602e55b13957df31e7350cbd55/packages/pictograms/metadata.yml). We don't manipulate the friendly name at all, so you'll wanna open a PR to update the mis-capitilized stuff in that file.

**Comment by connor-leech** [comment]
@vpicone My bad. I'll take a look a that.

**Comment by connor-leech** [comment]
@vpicone Fixing that yml will be pretty time-consuming and I likely won't have time to do it this week. I'll open an issue, but don't let that block you for now.

**Comment by vpicone** [comment]
Yeah I think they were written by hand so will likely need to be updated by hand as well.

**Comment by vpicone** [comment]
@chrisconnors-ibm those Icons weren't added correctly. There are no Milan or Mexico City icons, they are only wrappers for variants. They were given categories and aliases so the website is trying to look up an icon for them, those things need to be removed to match other cities/wrappers in the yml. This needs to get fixed upstream in Carbon.

Sidenote: the name for Milan Skyscrapers is just `Skyscrapers` for as opposed to the other `City Skyscrapers`

**Comment by vpicone** [comment]
@laurenmrice that's looks like it's a bug from us processing the svgs in Carbon. Not sure if we want to block this until it gets fixed. Opened an issue here: https://github.com/carbon-design-system/carbon/issues/4106

File: src/pages/guidelines/pictograms/contribute.mdx [hunk 1]
```diff
@@ -0,0 +1,94 @@
+---
+title: Pictograms
+description: Pictograms are visual symbols used to represent ideas, objects, or narratives at a glance. They work well in presentations and marketing communications.
+tabs: ['Library', 'Usage', 'Contribute']
+---
+
+<PageDescription>
+
+IBM’s pictograms are visual symbols used to represent ideas, objects, or
```
Comment (connor-leech) [inline src/pages/guidelines/pictograms/contribute.mdx]:
```suggestion
Pictograms are visual symbols used to represent ideas, objects, or
```

Comment (vpicone) [reply src/pages/guidelines/pictograms/contribute.mdx]: This content is copied from the IBM design language website so we'll need to make changes there.

**Comment by vpicone** [comment]
@laurenmrice @chrisconnors-ibm both issues have been resolved

**Comment by vpicone** [comment]
@laurenmrice fixed

**Comment by vpicone** [comment]
@laurenmrice that’s an issue with the metadata upstream in carbon. Can you make an issue there? There’s other issues with the metadata capitalization @connor-leech has opened an issue to address.

**Comment by chrisconnors-ibm** [comment]
@laurenmrice you're exactly right. If @connor-leech is making other metadata changes in another issue, that'd be a swell place to correct `Life Sciences` should be `Life Science`.

The risks of managing directly yaml values that are supposed to be generated from `categories.yml`

**Pull request merged**